lib/index.ts

```diff
@@ -12,9 +12,9 @@ type PrivateSchema = Omit<
 >;
 
 interface CreateSchemaOptions {
-  basePath: string;
   datasource: PrismaDataSourceOptions;
   generator: PrismaGeneratorOptions | PrismaMultiGeneratorOptions;
+  basePath?: string;
 }
 
 export let schema: PrismaSchema | undefined;
```

Comment:
With your latest changes, this variable isn't being set anymore - meaning that `createMixin`, `createEnum`, and `createModel` from within `@/util/create` are unusable since they import and check that this variable isn't `undefined`.

If my understanding is correct, then it looks like this will cause issues when users use these functions and will also break the standard usage example.

Reply:
Let me have a look, ftr I'm adding snapshot tests and eslint checks so that regressions like this are easier to catch.

Reply:
@efreila Ah, didn't realize I removed the `schema` variable setting. Whew, try again after this recent fixup.

Reply:
No worries, glad we were able to catch it 😅